WebRoot/js/ajax/dwt/widgets/DwtDialog.js

```diff
@@ -62,6 +62,8 @@ DwtDialog = function(params) {
 	params.className = params.className || "DwtDialog";
 	this._title = params.title = params.title || "";
 
+	appCtxt.notifyZimlets("onDwtDialog", [this, params]);
```

Comment:
I found it would cause an error on 2FA setup page at login (when zimbraFeatureTwoFactorAuthRequired is true and user has not setup it)
It must be as follows. I will update line 65 and 540.
```suggestion
	window.appCtxt && appCtxt.notifyZimlets("onDwtDialog", [this, params]);
```

Reply:
fixed it